pallets/qa-tools/Cargo.toml

```diff
@@ -16,12 +16,14 @@ codec = { package = "parity-scale-codec", version = "3", default-features = fals
     "derive",
 ] }
 common = { path = "../../common", default-features = false }
+rand = { version = "0.8.5", default-features = false }
+rand_chacha = { version = "0.3.1", default-features = false }
 scale-info = { version = "2", default-features = false, features = ["derive"] }
 sp-std = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 frame-support = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 frame-system = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 assets = { path = "../assets", default-features = false }
-order-book = { path = "../../pallets/order-book", default-features = false }
+order-book = { path = "../../pallets/order-book", default-features = false, features = ["private-net"] }
```

Comment:
Looks redundant because you include order-book with `private-net` feature below. It should work if you build `qa-tools` with `private-net` feature.

Reply:
It could mean you have problems with feature flags in `qa-tools` pallet

Reply:
> It could mean you have problems with feature flags in `qa-tools` pallet

As far as I understand, `cargo check` includes `qa-tools` without private-net because it is included in `std` feature. This way (with `optional = true` set for the crate) it is included without the `private-net` feature, even though it remains unused. Not sure how to fix it, I can check when I have more time later;

Btw I'm always open to your suggestions to solve it as I'll be on a vacation for next 2 weeks :)

Reply:
I would fix it immediately, but I don't see any ways to fix it rn